app/controllers/hq/partners_controller.rb

```diff
@@ -5,4 +5,11 @@ class Hq::PartnersController < ApplicationController
   def index
     @partners= Partner.all
   end
+
+  def show
+    if not (@partner= Partner.find_by_id(params[:id]))
```

Comment:
@PurityControl created some nice methods in the controller, like `load_partner`. Would it be better to use those here too?
https://github.com/AgileVentures/osra/pull/252/files